.flake8

```
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E203, E266, W503, F403, F401
+ignore = E203, E266, W503, F403, F401, W605, E711
```

```suggestion
ignore = E203, E266, W503, F403, F401, E711
```

Лучше использовать raw string: https://www.flake8rules.com/rules/W605.html